test/unit/lib/plugins/aws/package/compile/events/eventBridge/index.test.js

```diff
@@ -127,6 +127,43 @@ const serverlessConfigurationExtension = {
         },
       ],
     },
+    retryPolicyConfiguration: {
```

Comment:
I think we could combine both into single case to avoid unnecessary double `runServerless` run, what do you think?